### PR TITLE
[ATIP] Update environment.py support in README.md

### PR DESCRIPTION
**Pull request by Honry**
Add specific environment.py for uiautomator only tests to eliminate dependency from WebDriver

### CODE REVIEW

**Comment by Honry** [comment]
@zqzhang, PTAL.

File: tools/atip/README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ ATIP(Application Test in Python), a "behave" binding library as the bridge betwe
 
 ## Configuration
 
-Environmental controls. In ATIP usage, you can write the environment initial processes in config file - environment.py. ATIP provides a template of "environment.py" for tests developers. This template supports running tests independently by "behave" tool.
+Environmental controls. In ATIP usage, you can write the environment initial processes on this file. ATIP provides two templates of "environment.py" for tests developers. One is for WebDriver backend, the other one is for uiautomator backend. These templates support running tests independently by "behave" tool.
```
Comment (zqzhang) [inline tools/atip/README.md]: s/the other one/the other/

File: tools/atip/README.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ In Crosswalk testing, the template need to know the some test vars, e.g. which d
   WEBDRIVER_VARS: webdriver_url, desired_capabilities, others  
   LAUNCHER: XWalkLauncher, CordovaLauncher
 
+In uiautomator testing, only two environment vars are required, "TEST_PLATFORM" and "DEVICE_ID". This allows running uiautomator tests independently without WebDriver.
```
Comment (zqzhang) [inline tools/atip/README.md]: better use variables rather than vars

**Comment by Honry** [comment]
Thanks for your comments, updated.

**Pull request merged**